docs-src/stargate-develop/modules/quickstart/pages/qs-document.adoc

```diff
@@ -56,5 +56,5 @@ include::develop:page$api-doc/doc-document-deleting.adoc[Deleting documents and
 
 Voila! For more information on the Document API, see the
 xref:develop:dev-with-doc.adoc[Using the Document API]
-or the https://stargate.io/docs/stargate/1.0/attachments/docv2.html[Document API] in the
+or the https://stargate.io/docs/latest/develop/attachments/docv2.html[Document API] in the
```

Comment:
@polandll how about making it like `xref:develop:api.adoc[Document API]` so we make folks to the API landing page where they come to know about other APIs too? Thoughts?

Reply:
Nah - it's easy to find the API references - it's right there in the lefthand nav. Let's just pop them here to the exact API.